CougSat1-IHU/src/systemInterfaces/PMIC.h

```diff
@@ -16,15 +16,92 @@
 
 #define SRC_SYSTEMINTERFACES_PMIC_H
 
+#define I2C_ADDR_PMIC 0x0E
+
 #include <mbed.h>
 #include <systemInterfaces/Subsystem.h>
 
 class PMIC : public SubSystem {
   public:
+    enum TargetSubsystem {
+      IHU       = 0x00,
+      IFJR      = 0x01,
+      ADCS      = 0x02,
+      ADCSCoils = 0x03,
+      Comms     = 0x04,
+      CommsAmp  = 0x05,
+      Payload0  = 0x06,
+      Payload1  = 0x07,
+      Payload2  = 0x08
+    };
+    enum TargetReading {
+      // voltage/current:
+      SolarPanel0 = 0x00,
+      SolarPanel1,
+      SolarPanel2, 
+      SolarPanel3,
+      SolarPanel4,
+      SolarPanel5,
+      SolarPanel6,
+      SolarPanel7,
+      BatteryA,
+      BatteryB,
+      RegulatorA3V3,
+      RegulatorB3V3,
+
+      // current:
+      PR_3V3_0,
+      PR_3V3_1,
+      PR_3V3_2,
+      PR_3V3_3,
+      PR_3V3_4,
+      PR_3V3_5,
+      PR_3V3_6,
+      PR_3V3_7,
+      PR_3V3_8,
+      PR_3V3_9,
+      PR_3V3_10,
+      PR_3V3_11,
+      PR_3V3_12,
+      PR_BATT_0,
+      PR_BATT_1,
+      PR_BATT_2,
+      PR_BATT_3,
+      PR_BATT_4,
+      PR_BATT_5,
+      PR_BATT_6,
+      PV_3V3_0,
+      PV_3V3_1,
+      PV_3V3_2,
+      PV_3V3_3,
+      PR_BH_0,
+      PR_BH_1,
+      PR_DEPLOY,
+
+      // for temp readings:
+      TargetPMIC,
```

Comment:
Capitalize your comments and use full words to remove ambiguity, they are sentences. Did you prefix PMIC with target just because or because there was conflicts with the class name?

Reply:
Yes. I did prefix PMIC with Target because it conflicted with the class name. I am unsure if there is a better way of resolving the conflict (perhaps moving the scope of the enum outside of the class might do it?).